### DISCUSSION TITLE
-- WARNING -- Error loading Podman deps Lib. Podman feature in the Containers plugin is disabled (No module named 'podman')

### DISCUSSION THREAD

**Discussion by henkiewie** [nicolargo/glances#2937] [Q&A] [answered]
Hi,

I've installed using the command `pip install --user 'glances[web]'` and the systemd service as described.

But the service won' start.
The log files says;
`-- WARNING -- Error loading Podman deps Lib. Podman feature in the Containers plugin is disabled (No module named 'podman')`

command line 'glances -w' starts like normal. Is there a way to solve this?

OS is KDE neon (ubuntu jammy)

It is working as expected on Linux mint (Wilma).

Greetings,
Henk

**Comment by RazCrimson** [comment]
This log message is unrelated to the server not starting up. Can you check and post systemd logs?

**Comment by henkiewie** [comment]
Hi,

It is the message in the log when starting. Which by the way is in my homedir? But its not a systemctl --user service.

Just now I moved to the docker (host) version and with restart-unless stopped it always restarts when I reboot.
So thats a good workaround.
When I do `systemctl status glances.service it just gives me this.

```
× glances.service - Glances
     Loaded: loaded (/etc/systemd/system/glances.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Sat 2024-09-07 07:19:04 CEST; 1 day 1h ago
    Process: 2597993 ExecStart=/usr/local/bin/glances -w (code=exited, status=1/FAILURE)
   Main PID: 2597993 (code=exited, status=1/FAILURE)
        CPU: 122ms

sep 07 07:19:04 dell-optiplex3060 systemd[1]: glances.service: Scheduled restart job, restart counter is at 5.
sep 07 07:19:04 dell-optiplex3060 systemd[1]: Stopped Glances.
sep 07 07:19:04 dell-optiplex3060 systemd[1]: glances.service: Start request repeated too quickly.
sep 07 07:19:04 dell-optiplex3060 systemd[1]: glances.service: Failed with result 'exit-code'.
sep 07 07:19:04 dell-optiplex3060 systemd[1]: Failed to start Glances.

```

This in the log when `journactl | grep glances`

```
sep 07 07:19:04 dell-optiplex3060 systemd[1]: glances.service: Main process exited, code=exited, status=1/FAILURE
sep 07 07:19:04 dell-optiplex3060 systemd[1]: glances.service: Failed with result 'exit-code'.
sep 07 07:19:04 dell-optiplex3060 systemd[1]: glances.service: Scheduled restart job, restart counter is at 4.
sep 07 07:19:04 dell-optiplex3060 glances[2597993]: Traceback (most recent call last):
sep 07 07:19:04 dell-optiplex3060 glances[2597993]:   File "/usr/local/bin/glances", line 5, in <module>
sep 07 07:19:04 dell-optiplex3060 glances[2597993]:     from glances import main
sep 07 07:19:04 dell-optiplex3060 glances[2597993]:   File "/usr/local/lib/python3.10/dist-packages/glances/__init__.py", line 36, in <module>
sep 07 07:19:04 dell-optiplex3060 glances[2597993]:     from glances.logger import logger
sep 07 07:19:04 dell-optiplex3060 glances[2597993]:   File "/usr/local/lib/python3.10/dist-packages/glances/logger.py", line 18, in <module>
sep 07 07:19:04 dell-optiplex3060 glances[2597993]:     from glances.globals import safe_makedirs
sep 07 07:19:04 dell-optiplex3060 glances[2597993]:   File "/usr/local/lib/python3.10/dist-packages/glances/globals.py", line 39, in <module>
sep 07 07:19:04 dell-optiplex3060 glances[2597993]:     from defusedxml.xmlrpc import monkey_patch
sep 07 07:19:04 dell-optiplex3060 glances[2597993]: ModuleNotFoundError: No module named 'defusedxml'

```
So definitly something wrong with the installation. Maybe because I tried the sudo apt version first.
For now I will stick with the docker version. And will give the uninstall script a try.
Thanks for trying to help.

Greetings Henk

**Reply by RazCrimson** [reply]
Yep, its crashing as its missing dependencies.

You used the install script to setup glances?

**Comment by henkiewie** [comment]
I did, and when i rerun its says requirement already satisfied on all entries.

**Comment by henkiewie** [comment, marked as answer]
I just ran uninstall script after cloning the repo. Then ran the curl command and the service is starting again.
